python/dolfinx/mesh.py

```diff
@@ -126,6 +126,32 @@ def MeshTags(mesh, dim, indices, values):
     return fn(mesh, dim, indices, values)
 
 
+def create_boundary_mesh(mesh, comm, orient=False):
```

Comment:
Let's put this back in the plotting module. We don't want to encourage its use without careful thought.